developers_chamber/slack_utils.py

```diff
@@ -0,0 +1,35 @@
+import subprocess
+
+from slack_sdk import WebClient
+
+
+def run_sh(cmd: str) -> list:
+    return [
+        line for line in subprocess.run(
+            cmd, shell=True, check=True, stdout=subprocess.PIPE
+        ).stdout.decode('utf-8').rstrip().split('\n')
+    ]
+
+
+def upload_new_migration(token, channel, diff_branch):
+    client = WebClient(token=token)
+    current_branch = run_sh('git branch --show-current')[0]
+
+    migration_files = run_sh(
```

Comment:
We already do have code that iterates migrations in diff...
https://github.com/druids/developers-chamber/blob/master/developers_chamber/qa/checks.py#L27

Reply:
Good point. I changed it.